Ports/RetroArch/package.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=RetroArch
+useconfigure="true"
+version="1.10.3"
+workdir="${port}-${version}"
+archive_hash="2af44294e55f5636262284d650cb5fff55c9070ac3a700d4fa55c1f152dcb3f2"
+files="https://github.com/libretro/${port}/archive/refs/tags/v${version}.tar.gz ${port}-${version}.tar.gz $archive_hash"
+auth_type=sha256
+depends=("freetype" "SDL2" "zlib")
+
+configopts=(
+    "--disable-builtinglslang"
+    "--disable-discord"
+    "--disable-glsl"
+    "--disable-glslang"
+    "--disable-opengl"
+    "--disable-slang"
+    "--disable-spirv_cross"
+    "--disable-update_cores"
+)
```

Comment:
Thank you, indeed there is linking issue when it is built with `mbedlts` from the port. I will add `--disable-systemmbedtls` so that we use the builtin from the deps directory.

<details>
  <summary>Linking issue</summary>

```
LD retroarch
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_init':
net_socket_ssl_mbed.c:(.text+0xd1): undefined reference to `mbedtls_net_init'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_connect':
net_socket_ssl_mbed.c:(.text+0x334): undefined reference to `mbedtls_net_recv'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: net_socket_ssl_mbed.c:(.text+0x33b): undefined reference to `mbedtls_net_send'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_receive_all_nonblocking':
net_socket_ssl_mbed.c:(.text+0x3ea): undefined reference to `mbedtls_net_set_nonblock'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_receive_all_blocking':
net_socket_ssl_mbed.c:(.text+0x482): undefined reference to `mbedtls_net_set_block'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_send_all_blocking':
net_socket_ssl_mbed.c:(.text+0x50f): undefined reference to `mbedtls_net_set_block'
/stockage/serenity/Toolchain/Local/i686/lib/gcc/i686-pc-serenity/11.2.0/../../../../i686-pc-serenity/bin/ld: obj-unix/release/./libretro-common/net/net_socket_ssl_mbed.o: in function `ssl_socket_send_all_nonblocking':
net_socket_ssl_mbed.c:(.text+0x58e): undefined reference to `mbedtls_net_set_nonblock'
collect2: error: ld returned 1 exit status
make: *** [Makefile:201 : retroarch] Erreur 1
```

</details>

Reply:
[Disable system mbedtls](https://github.com/SerenityOS/serenity/pull/13855/commits/b2e40281068f39110ebac164177d01b520105e01)